maven-artifact/src/main/java/org/apache/maven/artifact/handler/ArtifactHandler.java

```diff
@@ -19,10 +19,10 @@
 package org.apache.maven.artifact.handler;
 
 /**
- * An artifact handler defines for a dependency type, defined as Plexus role:<ul>
- * <li>extension and classifier, to be able to download the file,</li>
+ * An artifact handler contains metadata derived from the dependency element that references the artifact:<ul>
+ * <li>extension and classifier, to be able to download the file</li>
```

Comment:
`extension and classifier` -> `défaut extension and classifier`

Reply:
finally, it's more `extension and default classifier`